src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/helper.ts

```diff
@@ -952,3 +952,25 @@ export function buildPartialMatcher(str: string) {
   // Return the final regex pattern
   return new RegExp(pattern + '$', 'i');
 }
+
+const isNullMatcher = buildPartialMatcher('is nul');
+const isNotNullMatcher = buildPartialMatcher('is not nul');
+
+/**
+ * Encapsulates handling of the "is null" and "is not null"
+ * checks...
```

Comment:
Something missing here?

Reply:
Nope. That's all I have to say 🤣

Reply:
Updated in https://github.com/elastic/kibana/pull/216379/commits/866a5e41e331836150f1f575b1aaa279b2f0764f